pkg/services/sqlstore/migrations/live.go

```diff
@@ -0,0 +1,25 @@
+package migrations
+
+import (
+	. "github.com/grafana/grafana/pkg/services/sqlstore/migrator"
+)
+
+func addLiveMigrations(mg *Migrator) {
+	liveMessage := Table{
+		Name: "live_message",
+		Columns: []*Column{
+			{Name: "id", Type: DB_BigInt, Nullable: false, IsPrimaryKey: true, IsAutoIncrement: true},
+			{Name: "org_id", Type: DB_BigInt, Nullable: false},
+			{Name: "channel", Type: DB_NVarchar, Length: 189, Nullable: false},
+			{Name: "data", Type: DB_Text, Nullable: false},
+			{Name: "created_by", Type: DB_Int, Nullable: false},
+			{Name: "created", Type: DB_DateTime, Nullable: false},
```

Comment:
```suggestion
			{Name: "config", Type: DB_Text, Nullable: false}, // channel configuration
			{Name: "message", Type: DB_Text, Nullable: false}, // message that will be sent on subscribe
			{Name: "created", Type: DB_DateTime, Nullable: false},
			{Name: "updated", Type: DB_DateTime, Nullable: false},
```

I don't think we need `created_by`

Reply:
Removed it